### PR TITLE
EWPP-970: Add char limit to first name and last name fields.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

**Review by 22Alexandra — DISMISSED** [review]

**Review by sergepavle — DISMISSED** [review]

File: modules/oe_content_person/oe_content_person.post_update.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add length restrictions for title display fields.
+ */
+function oe_content_person_post_update_20004(): void {
+  // Update form display to add maxlength to first and last names.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_person') . '/config/post_updates/20001_person_title');
```
Comment (upchuk) [inline modules/oe_content_person/oe_content_person.post_update.php:17]: 20004 vs 20001

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by upchuk — APPROVED** [review]

File: modules/oe_content_person/oe_content_person.post_update.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add length restrictions for title display fields.
+ */
+function oe_content_person_post_update_20001(): void {
+  // Update form display to add maxlength to first and last names.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_person') . '/config/post_updates/20001_person_title');
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $display_values = $storage->read('core.entity_form_display.node.oe_person.default');
```
Comment (sergepavle) [inline modules/oe_content_person/oe_content_person.post_update.php:19]: We have to update not this config, but field.storage.

File: modules/oe_content_person/config/post_updates/20001_person_title/core.entity_form_display.node.oe_person.default.yml [hunk 3]
```diff
@@ -0,0 +1,400 @@
+langcode: en
```
Comment (sergepavle) [inline modules/oe_content_person/config/post_updates/20001_person_title/core.entity_form_display.node.oe_person.default.yml:1]: we need to update field.storage config

**Review by sergepavle — CHANGES_REQUESTED** [review]

File: modules/oe_content_person/oe_content_person.post_update.php (hunk 2 above)
Comment (imanoleguskiza) [reply modules/oe_content_person/oe_content_person.post_update.php:19]: It was decided together with business that for first and last name we don't update the storage, but rather only add the restriction in the form. If you see the ticket description it was updated to reflect this.

**Review by sergepavle — APPROVED** [review]

File: modules/oe_content_person/oe_content_person.post_update.php (hunk 2 above)
Comment (sergepavle) [reply modules/oe_content_person/oe_content_person.post_update.php:19]: I got it

**Pull request merged**